code/core/api/core.api

```diff
@@ -1030,6 +1030,28 @@ public final class com/adobe/marketing/mobile/services/ui/message/InAppMessageSe
 	public static fun values ()[Lcom/adobe/marketing/mobile/services/ui/message/InAppMessageSettings$MessageGesture;
 }
 
+public final class com/adobe/marketing/mobile/services/ui/pushtemplate/AEPPushPayload {
```

Comment:
What do you feel about renaming `pushtemplate` package to `notification`  -> `com.adobe.marketing.mobile.services.ui.notification`

Reply:
this makes sense as the code within can handle both types of notifications